### ISSUE TITLE
Make reading enchantment of lower levels

### ISSUE THREAD

**Issue by liach** [Nuchaz/BiblioCraft#468]
It always appear on lvl 30 enchantments with a very high chance. Can you reduce the max enchantability and increase its rarity?

**Comment by Nuchaz** [comment]
yeah, I'll look into doing that. I wanted it to be fairly rare anyway, I must have goofed that one up.
